AK/FixedPoint.h

```diff
@@ -15,6 +15,10 @@
 #ifndef KERNEL
 #    include <AK/Math.h>
 #endif
+#include <AK/UFixedBigInt.h>
+#include <AK/UFixedBigIntDivision.h>
```

Comment:
Did you mean to put this below that `#ifndef`?

Reply:
Oh yeah, was moved outside for debugging and I  apparently forgot to move it back in